test/recipes/80-test_cipherlist.t

```diff
@@ -20,7 +20,7 @@ setup("test_cipherlist");
 my ($build_version, $library_version) = openssl_versions();
 plan skip_all =>
     "This test recipe isn't supported when doing regression testing"
-    if $build_version != $library_version;
+    if $build_version ne $library_version;
```

Comment:
Is this a bug fix?

Reply:
Yes.  `test/versions.c` doesn't print the hex versions any more, so neither does `OpenSSL::Test::openssl_versions()`.